tasks/lookup.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Get manila access info
+  os_manila_share:
+    name: "{{ os_manila_mount_share_name }}"
+    user: "{{ os_manila_mount_share_user }}"
+    protocol: "{{ os_manila_mount_share_protocol }}"
+  environment:
+    OS_CLOUD: "{{ os_manila_mount_os_config_name | default(omit) }}"
```

Comment:
Unfortunately, this use of default(omit) does not work as intended - Ansible only supports omitting parameters between modules - it actually sets the parameter to some magic gobbledigook instead of not defining it.